### DISCUSSION TITLE
Is https://github.com/boto/botocore/pull/1810 in 1.24.89 yet? Or, when?

### DISCUSSION THREAD

**Discussion by ScottABrown** [boto/botocore#2776] [Q&A] [answered]
Is issue 1810 in anything up to and including boto3 1.24.89, the patest release on pypi as of this writing?

If not, is there any way to say when it will hit a general release on pypi?

Thanks,
-Scott

**Comment by tim-finnigan** [comment, marked as answer]
Hi Scott - this should be merged by now. I can see [this change](https://github.com/boto/botocore/blob/8cdfb7567b1f680e684186e50eb0b8c781a935b4/botocore/client.py#L865-L868) for example in the code. Which version are you using?
